shell/platform/embedder/embedder.cc

```diff
@@ -1054,6 +1125,9 @@ void PopulateSnapshotMappingCallbacks(
           args->isolate_snapshot_instructions,
           SAFE_ACCESS(args, isolate_snapshot_instructions_size, 0));
     }
```

Comment:
Maybe move this block to `PopulateAOTSnapshotMappingCallbacks`, makes it more readable.

Reply:
Makes sense. I restructure those functions.